frontend/src/metabase/admin/databases/components/DatabaseHelpSidePanel/EmbeddedEngineDocContent/constants.ts

```diff
@@ -0,0 +1,23 @@
+import type { EngineKey } from "metabase-types/api";
+
+// Maps engine key to the markdown file in docs/databases/connections/
```

Comment:
Prefer using JSDoc over comments so that people can benefit from better intellisense.